src/System.IO.Ports/tests/Legacy/SerialPort/Event_Generic.cs

```diff
@@ -152,7 +152,7 @@ public void EventHandler_ThrowsException()
         }
     }
 
-    [ConditionalFact(nameof(HasNullModem))]
+    [ConditionalFact(nameof(HasNullModem), Skip="Always hangs")]
```

Comment:
Should have an associated issue, we generally don't use Skip directly. The only exception would be a test that you're expected to manually run, which should have a detailed comment explaining that.